Resources/templates/responsive/invest/partials/reward_box_resign.php

```diff
@@ -1,5 +1,3 @@
-    <form class="form-horizontal" role="form" method="GET" action="/invest/<?= $this->project->id ?>/payment">
```

Comment:
What is the purpose of removing the `<form>` tag from this partial to be declared on insertion? Specially when this partial only holds form markup.

Also please, note the orphaned closing `</form>` tag left at https://github.com/GoteoFoundation/goteo/blob/d19276f610b4c3d76098a4f44df89099cedcd98c/Resources/templates/responsive/invest/partials/reward_box_resign.php#L23

If going ahead with decoupling `<form>` declaration from partial, I think it's best to also have to close the form markup after partial insertion for better code readability.